### ISSUE TITLE
[CSP, 2.6.3] NRE in NetworkTransform.sync___set_value__sendToOwner

### ISSUE THREAD

**Issue by AhonenAJ** [FirstGearGames/FishNet#169]
Hello!

I just updated to 2.6.3 Pro and there seems to be something funky going on. I assume it's related to: "Changed NetworkTransform SendToOwner is now a SyncVar." 2.6.2 works fine. No idea what's going on there since the issue is happening somewhere in the generated code and I can't debug it. Happens on both the host and the clients for all PredictedObject components in my scene and spawned prefabs. All the affected objects are using prediction via NetworkTransform, and none are referencing any null objects in the inspector. Other than this one error message, everything seems to be functioning normally.

```
NullReferenceException: Object reference not set to an instance of an object
FishNet.Component.Transforming.NetworkTransform.sync___set_value__sendToOwner (System.Boolean value, System.Boolean asServer) (at <47f5a88bb7924ce686117dbefd0f747a>:0)
FishNet.Component.Transforming.NetworkTransform.ConfigureForCSP () (at Assets/FishNet/Runtime/Generated/Component/NetworkTransform/NetworkTransform.cs:1910)
FishNet.Component.Prediction.PredictedObject.ConfigureNetworkTransform () (at Assets/FishNet/Runtime/Generated/Component/Prediction/PredictedObject.cs:541)
FishNet.Component.Prediction.PredictedObject.OnValidate () (at Assets/FishNet/Runtime/Generated/Component/Prediction/PredictedObject.cs:555)
```

**Comment by FirstGearGames** [comment]
Does this happen at runtime?

**Comment by FirstGearGames** [comment]
```cs
        internal void ConfigureForCSP()
        {
            if (Application.isPlaying)
                return;```
                
                Put the Application.isPlaying check at the top of ConfigureForCsp()

**Comment by FirstGearGames** [comment]
I'm going to make a quick release for this.

**Comment by FirstGearGames** [comment]
Resolved in 2.6.4
